.idea/gradle.xml

```diff
@@ -4,7 +4,7 @@
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
-        <option name="testRunner" value="PLATFORM" />
```

Comment:
Please remove the changes that arent related

Reply:
Noted. You mean the Non-Kotlin Files?